### PR TITLE
#163009012: Handle exception raised when internet fails

### PR DESCRIPTION
**Pull request by blac-siren**
#### What does this PR do?
Handle exception raised when internet fails. `ConnectionError`
#### How should this be manually tested?
- turn off your wifi and make any request whether  graphql or rest.
#### What are the relevant pivotal tracker stories?
[#163009012](https://www.pivotaltracker.com/story/show/163009012)
#### Screenshots (if appropriate)
_Rest_request
image
_Graphql_ request
image

### CODE REVIEW

File: helpers/auth/authentication.py [hunk 1]
```diff
@@ -144,4 +142,20 @@ def wrapper(*args, **kwargs):
         return decorator
 
 
+def failed_conn(*args):
```
Comment (Bonifase) [inline helpers/auth/authentication.py]: Use a more descriptive function definition like `def failed_connection`

Comment (blac-siren) [reply helpers/auth/authentication.py]: @Bonifase I've implemented your feedback.

**Comment by cwizard2011** [comment]
@joshuaocero I have implemented the feedback

**Comment by joshuaocero** [comment]
Nice! LGTM

**Pull request merged**